.github/ISSUE_TEMPLATE/qg-checklist.md

```diff
@@ -42,15 +43,16 @@ Release Managemnet Reference Issue: <!-- Note: Add the related product RM issue
 
 #### TRG 3 Kubernetes
 
-- [ ] [TRG 3.02](https://eclipse-tractusx.github.io/docs/release/trg-3/trg-3-2) persistent volume and persistent volume claim is used when needed
+- [ ] [TRG 3.02](https://eclipse-tractusx.github.io/docs/release/trg-3/trg-3-2) persistent volume and persistent volume claim or database dependency (subchart) are in place when needed
 
 #### TRG 4 Container
 
 - [ ] [TRG 4.01](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-01) [semantic versioning](https://semver.org/) and tagging <!-- container is tagged correctly additionally to the latest tag -->
 - [ ] [TRG 4.02](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-02) base image is agreed  <!-- Java, Kotlin, ... if JVM based language use base image from [Eclipse Temurin](https://hub.docker.com/_/eclipse-temurin) -->
 - [ ] [TRG 4.03](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-03) image has `USER` command and Non Root Container
-- [ ] [TRG 4.05](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-05) released image must be placed in `DockerHub`, remove `GHCR` references 
+- [ ] [TRG 4.05](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-05) released image must be placed in `DockerHub`, remove `GHCR` references
 - [ ] [TRG 4.06](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-06) separate notice file for `DockerHub` has all necessary information
+- [ ] [TRG 4.06] (https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-07) root filesystem is set to read only
```

Comment:
I would recommend to rephrase it like this:
```suggestion
- [ ] [TRG 4.06] (https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-07) The root file system is set to read access by default, but can be overwritten by the user
```

Reply:
> I would recommend to rephrase it like this:

thank you, done